src/Domain/RepositoryException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+/**
+ * Exception class that converts low-level storage layer exceptions to the domain.
+ *
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class RepositoryException extends \RuntimeException {
+	const CODE_DEFAULT = 0;
```

Comment:
The default already is 0. And if you want to have a new explicit default for this class, then I think it's better to do that in this class (ie by means of constructor).